litmus/PTX/Proxy-Const-CoRR-cause1.litmus

```diff
@@ -1,5 +1,6 @@
-PTX fence
-"Two thread example with a constant proxy fence"
+PTX Proxy-Const-CoRR-cause1
```

Comment:
Do we get the same result if we place the fence between the two stores instead of the two loads? This way we get the same pattern as in  `Proxy-Const-CoRR-cause-different-cta` but using the same cta

Reply:
Comment updated.